cli/src/stats_server.rs

```diff
@@ -0,0 +1,94 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+use clap::Args;
+use reqwest::{Client, Response};
+use serde_json::Value;
+
+use libparsec::{BackendAddr, DateTime};
+
+#[derive(Args)]
+pub struct StatsServer {
+    /// Server address (e.g: parsec://127.0.0.1:6770?no_ssl=true)
+    #[arg(short, long)]
+    addr: BackendAddr,
+    /// Administration token
+    #[arg(short, long)]
+    token: String,
+    /// Output format (json/csv)
+    #[arg(short, long, default_value_t = Format::Json)]
+    format: Format,
+    /// Ignore everything after this date (e.g: 2024-01-01T00:00:00-00:00)
+    #[arg(long)]
+    at: Option<DateTime>,
```

Comment:
I would prefer `start_date` as the doc-comment say it ignore everything before that date.

`at` could confuse the user that the stat will only be for a specific date

Reply:
Not before, after

Reply:
Maybe you prefer `ignore-after` ?

Reply:
Oops, `end-date` then

Reply:
Done